### DISCUSSION TITLE
How to upgrade calcom docker installation?

### DISCUSSION THREAD

**Discussion by davidwuest** [calcom/docker#158] [General] [answered]
What is the correct way to upgrade the calcom docker installation?

**Comment by krumware** [comment]
Hi! Pull the latest docker image, or build an updated image, check the environmental variables for updates or changes, and then run it. The database migrations will run on start.

**Comment by davidwuest** [comment, marked as answer]
@krumware Is there a pre build docker image?

**Reply by davidwuest** [reply]
Found it. Thanks, @krumware.

**Reply by davidwuest** [reply]
https://hub.docker.com/r/calendso/calendso
